src/engine/SCons/Script/Main.py

```diff
@@ -1255,7 +1255,11 @@ def order(dependencies):
 
 
     is_pypy = platform.python_implementation() == 'PyPy'
-    python_has_threads = sysconfig.get_config_var('WITH_THREAD') or is_pypy
+    # As of 3.7, python removed support for threadless platforms.
+    # See https://www.python.org/dev/peps/pep-0011/
+    is_37_or_later = sys.version_info.major > 3 or \
```

Comment:
You can write this test simpler:
```python
sys.version_info >= (3, 7)
```

Reply:
Ohh. I like that.

Reply:
@serhiy-storchaka  - I've merged this pull request. But I'm going to use your sys.version_info >= (3, 7).  Would you like some credit in the CHANGES.txt ? (If so, what name/identifier would you like used for such)?

Reply:
It was a trivial suggestion, and this is an idiomatic code. It is not worth mentioning in CHANGES.txt.

Reply:
@serhiy-storchaka - OK.  Thanks!